.gitignore

```diff
@@ -217,8 +228,7 @@ venv.bak/
 .dmypy.json
 dmypy.json
 
-#Database
-/db
```

Comment:
Why is the database path no longer ignored?

Reply:
@nlMeminger It is still ignored:

https://github.com/jwflory/django-rit-grasa/pull/66/files/69e11e94e3659973fc32c2c9a1b62f8423358461#diff-a084b794bc0759e7a6b77810e01874f2R5-R6

I regrouped our project-related paths together.